doc/introduction/interfaces/numpy.rst

```diff
@@ -110,14 +110,66 @@ with respect to both QNode parameters ``phi`` and ``theta``:
 
     phi = np.array([0.5, 0.1])
     theta = 0.2
-    dcircuit = qml.grad(circuit3, argnum=[0, 1])
+    dcircuit = qml.grad(circuit3)
 
 Evaluating this gradient function at specific parameter values:
 
 >>> dcircuit(phi, theta)
 (array([ -4.79425539e-01,   1.11022302e-16]), array(0.0))
 
 
+Non-differentiable arguments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note that **all positional arguments provided to the QNode are assumed to be differentiable
+by default**:
+
+>>> from pennylane import numpy as np
+>>> np.array([0.1, 0.2])
+tensor([0.1, 0.2], requires_grad=True)
+
+If you would like to provide explicit non-differentiable arguments to the
+QNode or gradient function, make sure to use a NumPy array that specifies
+``requires_grad=False``:
```

Comment:
```suggestion
If you would like to provide explicit non-differentiable arguments to the
QNode or gradient function, make sure to import numpy via ``from pennylane import numpy as np``
and create a NumPy array that specifies
``requires_grad=False``:
```

Reply:
Have rewritten this section to include your changes (and have added the import line to the examples to make it explicit!)